models/organization/org_times.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package organization
+
+import (
+	"code.gitea.io/gitea/models/db"
+	"xorm.io/builder"
```

Comment:
blank line.

Reply:
I'm sorry I don't understand :) Does it *need* a blank line? (Where?) Does it *have* an unnecessary blank line?

(`lint` and `gofrumpt` does not complain.)

Reply:
Generally as a convention in this code base we group imports in three blocks

```go
import (
    // stdlib

    // packages in this repo

    // packages from other modules
)
```

So a blank line between these imports would be our convention. 🙂

Reply:
Ah, thank you! I was not aware of the _second_ blank line.